build.gradle

```diff
@@ -136,6 +138,8 @@ task downloadAndInstallJRuby(dependsOn: verifyFile, type: Copy) {
     into "${projectDir}/vendor/jruby"
 }
 
+def jrubyBin = './vendor/jruby/bin/jruby' + (SystemUtils.IS_OS_WINDOWS ? '.bat' : '')
```

Comment:
I really like commons lang, but in this case it is probably unnecessary, this check can be done with a System property and avoid a dependency.

Reply:
@jakelandis true :) sec adjusting